pkg/kv/kvserver/rangefeed/unbuffered_registration.go

```diff
@@ -223,9 +223,13 @@ func (ubr *unbufferedRegistration) IsDisconnected() bool {
 //
 // nolint:deferunlockcheck
 func (ubr *unbufferedRegistration) runOutputLoop(ctx context.Context, forStacks roachpb.RangeID) {
+	start := timeutil.Now()
 	ubr.mu.Lock()
-	// Noop if publishCatchUpBuffer below returns no error.
-	defer ubr.drainAllocations(ctx)
+	defer func() {
+		// Noop if publishCatchUpBuffer below returns no error.
+		ubr.drainAllocations(ctx)
+		ubr.metrics.RangefeedOutputLoopNanos.Inc(timeutil.Since(start).Nanoseconds())
```

Comment:
If we are only incrementing this for unbuffered registrations. Let's make sure we make that clear in the name and description. 

Also, lets use crtime here just for consistency.

Reply:
Done. While making the changes, I thought it would also be useful to track how long each rangefeed lasts. I opened a separate PR for that since this patch is pretty large - https://github.com/cockroachdb/cockroach/pull/142134.